Cargo.toml

```diff
@@ -1,4 +1,3 @@
-resolver = "2"
```

Comment:
I'd definitely prefer we keep this, is there a reason it's being removed?

Reply:
So do I. It's placed outside of `[workspace]` here and amounts to nothing like this, so it should go back under `[workspace]`.

Reply:
Oops, mistake, yes let's leave it.